### PR TITLE
Suggest alternatig time for .NET SIG meetings

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Mornings do not work well for me and I suggest start alternating time for the SIG meeting.

@open-telemetry/dotnet-approvers  @open-telemetry/dotnet-maintainers do you have objections?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-
 
 ### .NET SDK
 
-Regular sync up will be held initially weekly - every Tuesday at 9 AM PDT. Check
+Regular sync up held weekly at Tuesdays at alternatig time - 9 AM PT and 4PM PT. Check
```
Comment (cijothomas) [inline README.md:153]: typo: alternating

Comment (SergeyKanzhelev) [reply README.md:153]:
```suggestion
Regular sync up held weekly at Tuesdays at alternating time - 9 AM PT and 4PM PT. Check
```

**Comment by MikeGoldsmith** [comment]
I don't have an objection to alternating weeks but I will not be able to attend 4pm Pacific Time as that's 12am (midnight) UK time.

The 9am (5pm UK) meeting is sometimes difficult for me to attend too because of commitments with kids. Ideally for me, I prefer before 9am PT (5pm UK) or between 11am and 1pm PT (7pm and 9pm).

Would a mid-morning PT slot work everyone?
eg 11:00 or 11:30am PT (that's 7:00 or 7:30 pm UK)

I've created this [Doodle](https://doodle.com/poll/t35vkbra9t7kg56x) to help coordinate what times work for people. Let me know if you'd like more options to be added.

File: README.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-
 
 ### .NET SDK
 
-Regular sync up will be held initially weekly - every Tuesday at 9 AM PDT. Check
+Regular sync up held weekly at Tuesdays at alternating time - 9 AM PT and 4PM PT. Check
```
Comment (MikeGoldsmith) [inline README.md:153]:
```suggestion
Regular sync up held weekly on Tuesdays at alternating time - 9 AM PT and 4PM PT. Check
```

**Comment by SergeyKanzhelev** [comment]
@MikeGoldsmith not too many replies on the Doodle. For me there is no difference between 9AM and 11AM - for the quarantine time neither works well. So perhaps we can reschedule to 11 so it will work for you better. Let's bring it back again tomorrow at the regular time to collect more replies.

**Comment by pjanotti** [comment]
All times on the Doodle are fine for me, any time between 07:00 AM 11:00 PM Pacific time are fine for me. 
@SergeyKanzhelev @MikeGoldsmith

**Comment by MikeGoldsmith** [comment]
Thanks @SergeyKanzhelev. Of the people who's contributed to the Doodle, everyone other than yourself is free at 11am PST and 3pm PST.

In that case, I think it would be fair to move to a two weekly schedule to accommodate everyone, 11am one week and 3pm the following week. We can re-evaluate again once schedules are less constrained by global events. Other SIGs are doing this to allow for better geographical coverage too, so may be something we stick with long term to improve accessability.

File: README.md [hunk 3]
```diff
@@ -150,7 +150,7 @@ You can also join us on [gitter](https://gitter.im/open-telemetry/opentelemetry-
 
 ### .NET SDK
 
-Regular sync up will be held initially weekly - every Tuesday at 9 AM PDT. Check
+Regular sync up held weekly on Tuesdays at alternating time - 9 AM PT and 4PM PT. Check
```
Comment (MikeGoldsmith) [inline README.md:153]:
```suggestion
Regular sync up held weekly on Tuesdays at alternating time - 11 AM PT and 4PM PT. Check
```

**Comment by SergeyKanzhelev** [comment]
Calendar is updated

**Comment by MikeGoldsmith** [comment]
Thanks @SergeyKanzhelev :+1:

**Comment by MikeGoldsmith** [comment]
@SergeyKanzhelev looks like the change to 4pm has been made, but the earlier time is still 9am PST. Please can you update to 11am PST.

**Comment by SergeyKanzhelev** [comment]
Should be fixed now. I'm not sure how it happened, sorry for confusion

**Comment by MikeGoldsmith** [comment]
I can see it has updated now - thanks.

**Pull request merged**